tests/pr_test.go

```diff
@@ -113,3 +112,45 @@ func TestRunAdvancedExampleUpgrade(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func setupOptionsStandardSolution(t *testing.T, prefix string) *testhelper.TestOptions {
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
+		Prefix:        prefix,
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"existing_kms_instance_crn": permanentResources["hpcs_south_crn"],
+		"kms_endpoint_type":         "public",
+		"resource_group_name":       options.Prefix,
+	}
+
+	return options
+}
+
+func TestRunStandardSolution(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsStandardSolution(t, "redis-st-da")
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunStandardUpgradeSolution(t *testing.T) {
+	t.Parallel()
+	t.Skip()
```

Comment:
We will need to rememeber to come back and remove this... or maybe skip upgrade test since neither the main module or existing sub modules are touched by this change.

Reply:
remove the skip and skip upgrade test in the commit message so we dont forget. There is no risk in skipping since DA does not yet exist, and we are not changing anything in the module

Reply:
addressed